src/iocore/dns/CMakeLists.txt

```diff
@@ -18,4 +18,15 @@
 add_library(inkdns STATIC DNS.cc DNSConnection.cc DNSEventIO.cc Inline.cc SplitDNS.cc)
 add_library(ts::inkdns ALIAS inkdns)
 
-target_link_libraries(inkdns PUBLIC ts::inkevent ts::inkhostdb ts::proxy ts::tscore)
+target_include_directories(
+  inkdns
+  PUBLIC "${PROJECT_SOURCE_DIR}/include"
+  PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${PROJECT_SOURCE_DIR}/src/iocore/net"
```

Comment:
Is this include into src/iocore/net necessary?

Reply:
It's truthful. `dns` includes some of inknet's `P_` headers.

Reply:
Now that the `P_` includes are relative this should not be necessary anymore.

Reply:
They don't look like they're relative to the project root. This might be a little shaky.